doc/under_sampling.rst

```diff
@@ -214,11 +214,11 @@ the samples of interest in green.
    :scale: 60
    :align: center
 
-The parameter ``ratio`` control which sample of the link will be removed. For
-instance, the default (i.e., ``ratio='auto'``) will remove the sample from the
-majority class. Both samples from the majority and minority class can be
-removed by setting ``ratio`` to ``'all'``. The figure illustrates this
-behaviour.
+The parameter ``sampling_target`` control which sample of the link will be
+removed. For instance, the default (i.e., ``sampling_target='auto'``) will
+remove the sample from the majority class. Both samples from the majority and
```

Comment:
I would change `remove the sample from` for `remove samples from`.  + I don't really understand what `control which sample of the link will be removed` means. `link` confuses me